src/components/controls3d/fps.js

```diff
@@ -0,0 +1,431 @@
+goog.provide('fps');
+
+
+
+/**
+ * @param {!Cesium.Scene} scene
+ * @param {!Angular.Scope} scope
+ * @constructor
+ */
+function FPS(scene, scope) {
```

Comment:
Shouldn't it be a service ? I'm pretty sure we will have the need to know which state is the FPS in other directives in the future.